### DISCUSSION TITLE
Custom Column Header is showing as Object Object

### DISCUSSION THREAD

**Discussion by niikkiin** [TanStack/table#3921] [Q&A]
Hi everyone, can you help me debug this issue? I've been trying to fix it since last week.

The column header I used here is some sort of like this but it is showing as `object object`. Is this an issue in v7? My current version is ` "@types/react-table": "^7.7.12"` and `"react-table": "^7.7.0",`.

```
      Header: () =>
        (
          <span>
            <i className="fa-tasks" /> Progress
          </span>
        ) as any,
```

![image](https://user-images.githubusercontent.com/26428605/168510002-ee26c919-575e-48fb-8d20-19ad61d2ca90.png)

Thank you guys!

**Comment by malikk908** [comment]
did you find any solution?
